src/mjolnir.py

```diff
@@ -167,7 +166,7 @@ def enable_mjolnir(self, admin_access_token: str) -> None:
             USERNAME,
             True,
             admin_access_token,
-            str(self._charm_state.synapse_config.server_name),
+            str(charm_state.synapse_config.server_name),
```

Comment:
Out of curiosity, is `str()` cast required? I think this field is already of str type :D

Reply:
Please feel free to resolve this in follow up PR since it seems that it has good amount of approvals already :)

Reply:
I will solve it in a next PR as it it not related to this PR. Thanks for the input!